src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -133,8 +139,34 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
-@include large-size {
-    .bdl-CollapsibleSidebar-logo {
-        margin-left: $collapsible-side-bar-h-padding + $bdl-grid-unit * 2 - 1px /* border */;
+@include breakpoint($medium-screen) {
+    .Body.is-responsive-web {
```

Comment:
Just checking, do we intend to remove `.is-responsive-web` in the future?

Also, is `.Body` necessary? Could it be removed for less specificity?

Reply:
@tjuanitas I also followed convention with the `.Body.is-responsive-web`

At some point we’ll want to rollout responsiveness as default. So the .body.is-responsive-web is not really meant to stay forever. We would clean it up like the .body.deja-blue at some point.